localization/languages/fr.json

```diff
@@ -0,0 +1,9 @@
+{
+  "translation": {
+    "48-text-0": "Ce texte sera échangé automatiquement",
```

Comment:
While it's the literal translation of "swap", I feel like `modifié` (modified) sounds better in French in this context